mmf/configs/defaults.yaml

```diff
@@ -83,12 +83,17 @@ training:
 
     # Should a lr scheduler be used
     lr_scheduler: false
+
+    # DEPRECATED: Use MMFScheduler directly instead
```

Comment:
Is MMFScheduler added in some earlier PR?

Reply:
Updated.